### ISSUE TITLE
Project Proposal Feedback

### ISSUE THREAD

**Issue by stephenjarrell19** [COGS118A/Group003-Sp22#3]
Total Points: 8/8

Title & Abstract (1/1)
Likely restructure your proposal; alot of the information in the abstract perhaps belongs in the data section, metric section, or problem statement section. 

You should also consider proofreading this proposal, as there are quite a few large (yet forgivable) mistakes that make it difficult to understand.

Try to be concise and specific in your abstract and other sections.

Background (1/1)
Nice!

Problem Statement (1/1)
Good job; this is formalized very well.

What techniques to balance class distribution?

What does "we will run this alongside our whole dataset" mean? Do you mean you are training a model and evaluating it on a test set of 400k unseen patients?

If you are using patient data, make sure your training set and test set does not include the same patient in your dataset.

Data (1/1)
Great job!

Proposed Solution (1/1)
Are you trying to choose the best model/algorithm? How will you accomplish this?

Metrics (1/1)
Well done.

Ethics & Privacy (1/1)
This is good.

Team Expectations + Timeline (1/1)
Also good.

You can reply to this feedback below. Contact me anytime if you want help improving your project or have any questions at all!

**Comment by Lord-of-Bugs** [comment]
Hi Stephen,
About our "Problem Statement" section, I don't believe we mentioned anything about "patient data" or "run this alongside our whole dataset". Can you please clarify?

**Comment by Lord-of-Bugs** [comment]
The abstract is now condensed and updated, credit to Boning.
See the most recent abstract and project progression in Checkpoint.

**Comment by stephenjarrell19** [comment]
@Lord-of-Bugs All I meant by this is that some medical datasets have multiple records of the same individual.

For example:
Maybe I visited the doctor 3 times for a checkup. That means there will be 3 rows of data that pertain to my personal combination of data features. If you train a machine learning model, you do not want to train on some of my data and test on the rest of it, because the information about my health will be included in the training AND test set, so when we test we will get an overestimation of our generalization performance. Does that make sense?

It's important to check by some unique identifier like patient ID, to verify that data from a patient is not present in both the training and test set

**Comment by Lord-of-Bugs** [comment]
@stephenjarrell19 
I see. Since it is now beyond the deadline for updating the checkpoint, we will pay attention to this during our next stage and reflect this update in our final pipeline
